sqlcipher_flutter_libs/README.md

```diff
@@ -1,7 +1,27 @@
 # sqlcipher_flutter_libs
 
-This package intentionally contains no Dart code. Flutter apps depending on this package will
+Flutter apps depending on this package will
 contain native `SQLCipher` libraries.
 
+### Problems on Android 6
+
+There appears to be a problem when loading native libraries on Android 6 (see [this issue](https://github.com/simolus3/moor/issues/895#issuecomment-720195005)).
+If you're seeing those crashes, you could try setting `android.bundle.enableUncompressedNativeLibs=false` in your `gradle.properties`
+file. Be aware that this increases the size of your application when installed.
+
+Alternatively, you can use the `applyWorkaroundToOpenSqlite3OnOldAndroidVersions` method from this library.
+It will try to open `sqlite3` in Java, which seems to work more reliably. After sqlite3 has been loaded from Java,
+we can open it in Dart too.
```

Comment:
```suggestion
It will try to open `sqlcipher` in Java, which seems to work more reliably. After the native library has been loaded from Java,
we can open it in Dart too.
```